examples/minio/05-ark-backupstoragelocation.yaml

```diff
@@ -26,5 +26,7 @@ spec:
     region: minio
     s3ForcePathStyle: "true"
     s3Url: http://minio.heptio-ark.svc:9000
+    # OR get minio URL per documentation (comment out previous line)
+    # s3Url: NODE_URL_OR_IP:NODE_PORT
```

Comment:
This needs to include `http://` or `https://` - if it's just `1.2.3.4:31234`, I'm pretty sure it won't work.